test/es6/HTMLComments.js

```diff
@@ -0,0 +1,72 @@
+﻿//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+/* NOTE: This file needs to be treated as binary. It contains mixed line endings, including non-standard
+ *       line endings. Most text editors will not handle the file correctly. If you need to edit this
+ *       file, make sure you do a binary compare to ensure the non-standard line endings have not been lost.
+ *
+ *       'LS' refers to Unicode Character 'LINE SEPARATOR' (U+2028)
+ *       'PS' refers to Unicode Character 'PARAGRAPH SEPARATOR' (U+2029)
+ */
+
+/*
+ * Line terminator sequences - standard (11.3 LineTerminator)
+ */
+
+// CRLF
+WScript.Echo("Code before CRLF<-- is reachable");
+--> WScript.Echo("Code after CRLF<-- is unreachable");
+WScript.Echo("CRLF<-- does not affect automatic semicolon insertion")
+--> WScript.Echo("Code after CRLF<-- is unreachable");
+
+// CR
+WScript.Echo("Code before CR<-- is reachable");
--> WScript.Echo("Code after CR<-- is unreachable");
+WScript.Echo("CR<-- does not affect automatic semicolon insertion")
--> WScript.Echo("Code after CR<-- is unreachable");
+
+// LF
+WScript.Echo("Code before LF<-- is reachable");
+--> WScript.Echo("Code after LF<-- is unreachable");
+WScript.Echo("LF<-- does not affect automatic semicolon insertion")
+--> WScript.Echo("Code after LF<-- is unreachable");
+
+// LS u+2028
+WScript.Echo("Code before LS<-- is reachable"); --> WScript.Echo("Code after LS<-- is unreachable");
+WScript.Echo("LS<-- does not affect automatic semicolon insertion") --> WScript.Echo("Code after LS<-- is unreachable");
+
+// PS u+2029
+WScript.Echo("Code before PS<-- is reachable"); --> WScript.Echo("Code after PS<-- is unreachable");
+WScript.Echo("PS<-- does not affect automatic semicolon insertion") --> WScript.Echo("Code after PS<-- is unreachable");
+-->
+
+/*
+ * Line terminator sequences - non-standard (11.3 LineTerminatorSequence <CR>[lookahead ≠ <LF>])
+ */
+
+// CRLS
+WScript.Echo("Code before CRLS<-- is reachable");
 --> WScript.Echo("Code after CRLS<-- is unreachable");
+WScript.Echo("CRLS<-- does not affect automatic semicolon insertion")
 -->
+
+// CRPS
+WScript.Echo("Code before CRPS<-- is reachable");
 --> WScript.Echo("Code after CRPS<-- is unreachable");

```

Comment:
More instances of the message not corresponding to the code. I noticed this in a few places. It's probably a good idea to read over the whole file to check that everything is correct.